cypress-tests/cypress/support/commands.js

```diff
@@ -1057,10 +1119,17 @@ Cypress.Commands.add(
     res_data,
     confirm,
     auto_fulfill,
-    globalState,
+    token,
+    globalState
   ) => {
-    for (const key in req_data) {
-      createConfirmPayoutBody[key] = req_data[key];
+    if (token === true) {
+      createConfirmPayoutBody.payout_token = globalState.get("paymentToken");
+      createConfirmPayoutBody.payout_type = req_data.payout_type;
+      createConfirmPayoutBody.priority = req_data.priority;
```

Comment:
Have this as seperate flow , lets call it `confirmPayoutWithToken`, have a separate config for this as well